### ISSUE TITLE
Go through website design process with Orgnizer

### ISSUE THREAD

**Issue by hwaldstein** [MatthewMeacham/HomeworkOrganizer#54]
I think the usability of our website is suffering slightly because we're not using the website design tools that we learned during GET HIP. I feel like there are a small handful of things that we would do differently if we used to design process to make changes, and I think we should consider switching back to the design process that we learned.

**Comment by MatthewMeacham** [comment]
I completely agree with this, I will ask my family to go through the website and I will observe their actions to see how they behave and I will report that to y'all

**Comment by hwaldstein** [comment]
Sounds good. I'll see if my family is willing to do the same.
